src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resObj = {};
```

Comment:
[CODE STYLE] - Avoid variable namings like `array` , `object` `arr`, `obj`, `result`. It will decrease code readability. Consider using a more descriptive variable name for `resObj` that reflects its purpose, such as `styleObject` or `cssProperties`.